package.json

```diff
@@ -18,7 +18,7 @@
   "licenses": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "basisjs": "basisjs/basisjs#1.5",
+    "basisjs": "basisjs/basisjs#v1.5.1",
```

Comment:
Because `1.5` wasn't released at the moment `package.json` was created. `basisjs/basisjs#1.5` means branch that doesn't exist now.